docs/multi-stage-query/msq-api.md

```diff
@@ -652,25 +338,25 @@ print(response.text)
 
 ```
 {
-    "task": "query-a6b65442-f77e-44e4-af28-ab3b711a27ac",
-    "status": {
-        "id": "query-a6b65442-f77e-44e4-af28-ab3b711a27ac",
-        "groupId": "query-a6b65442-f77e-44e4-af28-ab3b711a27ac",
-        "type": "query_controller",
-        "createdTime": "2022-07-27T20:09:23.551Z",
-        "queueInsertionTime": "1970-01-01T00:00:00.000Z",
-        "statusCode": "SUCCESS",
-        "status": "SUCCESS",
-        "runnerStatusCode": "WAITING",
-        "duration": 136636,
-        "location": {
-            "host": "ip-10-201-5-81.ec2.internal",
-            "port": -1,
-            "tlsPort": 8100
-        },
-        "dataSource": "kttm_simple",
-        "errorMsg": null
-    }
+  "task": "query-3dc0c45d-34d7-4b15-86c9-cdb2d3ebfc4e",
+  "status": {
+    "id": "query-3dc0c45d-34d7-4b15-86c9-cdb2d3ebfc4e",
+    "groupId": "query-3dc0c45d-34d7-4b15-86c9-cdb2d3ebfc4e",
+    "type": "query_controller",
+    "createdTime": "2022-09-14T22:12:00.183Z",
+    "queueInsertionTime": "1970-01-01T00:00:00.000Z",
+    "statusCode": "RUNNING",
+    "status": "RUNNING",
+    "runnerStatusCode": "RUNNING",
+    "duration": -1,
+    "location": {
+      "host": "localhost",
+      "port": 8100,
+      "tlsPort": -1
+    },
+    "dataSource": "kttm_simple",
+    "errorMsg": null
+  }
 }
 ```
 
```

Comment:
should line 382 / 389 be
```
GET /druid/indexer/v1/task/<taskId>/reports
```
Python example uses reports. Report singular 404 for me

Reply:
fixed.